src/jit/flowgraph.cpp

```diff
@@ -17719,6 +17724,52 @@ void Compiler::fgSetTreeSeqHelper(GenTreePtr tree, bool isLIR)
         return;
     }
 
+    // Special handling for dynamic block ops.
+    if (((tree->OperGet() == GT_ASG) && (tree->gtGetOp1()->OperGet() == GT_DYN_BLK)) ||
+        (tree->OperGet() == GT_STORE_DYN_BLK))
+    {
```

Comment:
Actually, there *is* an `OperIsDynBlkOp`, but apparently I didn't use it.

---
In reply to: [77256474](https://github.com/dotnet/coreclr/pull/7027#discussion_r77256474) [](ancestors = 77256474)